loader/loader.c

```diff
@@ -492,6 +492,12 @@ static bool loaderAddJsonEntry(const struct loader_instance *inst,
                                LPSTR json_path,    // JSON string to add to the list reg_data
                                DWORD json_size,    // size in bytes of json_path
                                VkResult *result) {
+
+    // Check for and ignore duplicates.
+    if (*reg_data && strstr(*reg_data, json_path)) {
+        return true;
```

Comment:
Ideally the paths should be transformed into canonical absolute pathnames before deduplication.

Reply:
That would be ideal, but I don't think the loader does it elsewhere. It is unlikely that this will cause trouble, though, since I would imagine the driver would install the exact same string for each DriverStore.

Reply:
I'm willing to give on this one. You can mark it resolved without further code change. Maybe add a code comment here explaining that we know it is not being done, that might help someone debugging a future problem should it ever become a problem.